views/js/back.js

```diff
@@ -91,6 +91,28 @@ $(window).ready(function () {
         });
     });
 
+    // Tab Content : Add
+    $(document).on('click', '.psre-add', function () {
+      $('.landscape').show();
+
+      $('#reminder_listing').removeClass('active').addClass('inactive');
+      $('#blockDisplay').removeClass('inactive').addClass('active');
+      $('.show-rea-block').removeClass('active').addClass('inactive');
+
+      $('.panel-body-').removeClass('inactive').addClass('active');
+      $('#saveContentConfiguration').attr('data-id', '');
+
+      $('.limit_text:visible').text($('.show-rea-block.active .content_by_lang:visible input[type="text"]').val().length);
+      $('.limit_description:visible').text($('.show-rea-block.active .content_by_lang:visible textarea').val().length);
+
+      var landscape = $('.panel-body- .psr-picto').attr('src');
+      if (typeof landscape === 'undefined' || landscape === undefined) {
```

Comment:
I'm wondering, isn't `typeof landscape === 'undefined'` enough?

Reply:
@jolelievre That permits to detect that the source exists and is not empty.

Reply:
Yep I know but I thought just testing `typeof landscape === 'undefined'` was enough

Reply:
Oh ! ... Modified !